examples/terraform/aws-dualstack/credentials_ipv6+ipv4.yaml

```diff
@@ -0,0 +1,5 @@
+cloudConfig: |
```

Comment:
We need some documentation regarding how this file is supposed to be used. You can update the README file for those configs for now.

Reply:
done